include/glow/Graph/Graph.h

```diff
@@ -573,18 +573,30 @@ class Function final : public Named {
                                                unsigned_t axis);
 
   /// Implements a batched reduce mean of the \p batch on the provided \p axis
-  /// with output type \p outTy with three nodes: a BatchedReduceAdd followed by
-  /// a DivNode with a SplatNode of the length of the \p axis
-  /// dimension. \returns the final DivNode.
-  DivNode *createBatchedReduceMean(llvm::StringRef name, TypeRef outTy,
-                                   NodeValue batch, unsigned_t axis);
-
-  /// Implements a batched reduce mean of the \p batch on the provided \p axis
-  /// with three nodes: a BatchedReduceAdd followed by a DivNode with a
-  /// SplatNode of the length of the \p axis dimension. \returns the final
-  /// DivNode.
-  DivNode *createBatchedReduceMean(llvm::StringRef name, NodeValue batch,
-                                   unsigned_t axis);
+  /// with output type \p outTy with nodes: a) AveragePool, or b) three nodes:
```

Comment:
This comment now seems wrong. I would just say it creates and returns a `BatchedReduceMeanNode`. If you really want to you could mention that it may be lowered to three three nodes, or may be optimized into an AvgPool, but I don't think it's necessary.